.gitignore

```diff
@@ -41,3 +41,4 @@ Thumbs.db
 /sonar-runner/build
 
 functional-tests/\.java-version
+package-lock.json
```

Comment:
This file is actually intended to be committed to source control. It ensures that regardless of the versioning in `package.json` that it can create the exact dependency tree that is used.

Reply:
I can add it back in. I generally remove it if it hasn't been altered for correcting audit/package issues as they tend to get long, but if we want to leave it in as a general rule, I'm happy to unignore it.

Reply:
Maybe worth checking with the team?

Reply:
Yeah, sounds good. I'll leave it in for now, and we can come up with a policy.